pkgs/applications/misc/wlclock/default.nix

```
@@ -0,0 +1,46 @@
+{ pkgs
+, stdenv
+, lib
+, fetchFromSourcehut
+, meson
+, ninja
+, cmake
+, pkg-config
+, wayland-protocols
+, wayland
+, cairo
+, scdoc
+}:
+stdenv.mkDerivation rec {
+  pname = "wlclock";
+  version = "1.0.1";
+
+  src = fetchFromSourcehut {
+    owner = "~leon_plickat";
+    repo = "wlclock";
+    rev = "v${version}";
+    sha256 = "sha256-aHA4kXHYH+KvAJSep5X3DqsiK6WFpXr3rGQl/KNiUcY=";
+  };
+
+  nativeBuildInputs = with pkgs; [
+    meson
+    ninja
+    cmake
+    pkg-config
+  ];
+
+  buildInputs = with pkgs; [
+    wayland-protocols
+    wayland
+    cairo
+    scdoc
+  ];
+
+  meta = with lib; {
+    description = "A digital analog clock for Wayland desktops.";
```

```suggestion
    description = "A digital analog clock for Wayland desktops";
```
another nitpick, sorry